### PR TITLE
added datagen for item and block models, and blockstates.

### PR DESCRIPTION
**Pull request by GideonHead**

### CODE REVIEW

File: src/main/java/com/mraof/minestuck/data/MinestuckData.java [hunk 1]
```diff
@@ -63,6 +63,9 @@ public static void gatherData(GatherDataEvent event)
 		
 		gen.addProvider(event.includeServer(), new StartingModusProvider(gen, Minestuck.MOD_ID));
 		
+		gen.addProvider(true, new MinestuckBlockStateProvider(gen, fileHelper));
+		gen.addProvider(true, new MinestuckItemModelProvider(gen, fileHelper));
```
Comment (kirderf1) [inline src/main/java/com/mraof/minestuck/data/MinestuckData.java:67]:
```suggestion
		gen.addProvider(event.includeClient(), new MinestuckBlockStateProvider(gen, fileHelper));
		gen.addProvider(event.includeClient(), new MinestuckItemModelProvider(gen, fileHelper));
```

**Review by kirderf1 — COMMENTED** [review]
Should be a good base for future use. Though the two new classes need a little bit or reformatting to use the same style as the rest of the project.

**Comment by GideonHead** [comment]
Done, what other changes need to be made?

**Comment by GideonHead** [comment]
I did some reformatting, lmk if any more needs to be done

**Comment by kirderf1** [comment]
That was reformatting in the wrong direction as the brace style is now specifically what we're not using (we have braces on new lines). Make sure that you're using the code style configuration included in the project when you reformat.

**Comment by GideonHead** [comment]
ah alright, I moved the braces to the new lines

**Comment by GideonHead** [comment]
alright all of the formatting issues are fixed with intellij auto-reformat, so I think this is finished

**Review by kirderf1 — APPROVED** [review]
Should be good like this. I suppose you could also convert more blocks and items to use it, but I also think it's fine to leave it for the future on a on-need-basis.

**Pull request merged**